### PR TITLE
{lang}[GCCcore/13.2.0] Python v3.11.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
fair-mastodon-c7g-4xlarge-0001 - Linux Rocky Linux 8.7, AArch64, ARM UNKNOWN (neoverse_n1), Python 3.6.8
See https://gist.github.com/boegel/242d8088d0c198416a183db322efb654 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3157.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/91508b5c050b2240fbb1a5f4c0810b8a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
easybuild2.novalocal - Linux CentOS Stream 8, POWER, IBM pSeries (emulated by qemu) (power9le), Python 3.6.8
See https://gist.github.com/boegel/71ff92225f6e3055d3fa77effc964531 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/Python/Python-3.11.5-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+}
+
+# order is important!
+# package versions updated 2023-05-17
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/p/Python/Python-3.11.5-GCCcore-13.2.0.eb:42]:
```suggestion
# package versions updated 2023-09-28
```

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
zen2-ubuntu-2304 - Linux Ubuntu 23.04 (Lunar Lobster), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.11.2
See https://gist.github.com/SebastianAchilles/7ee23204c314515b29584245f8d992a6 for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @boegel!

**Pull request merged**